deps/http-codec.lua

```diff
@@ -251,7 +251,11 @@ local function decoder()
     local len, term
     len, term = match(chunk, "^(%x+)(..)")
     if not len then return end
-    assert(term == "\r\n")
+    if term ~= "\r\n" then
+      if #chunk < 34 then return end
```

Comment:
34 ?

Reply:
32 hex digits + '\r\n'. It's an arbitrary limit, wasn't sure what to go with.

Reply:
would 18 make sense? `FFFFFFFFFFFFFFFF` (16 characters + 2 for \r\n) is the max int for len.

Reply:
Yeah, that would probably make more sense.